charts/quickwit/values.yaml

```diff
@@ -159,7 +262,43 @@ config:
   # searcher:
   #   fast_field_cache_capacity: 10G
   #   split_footer_cache_capacity: 1G
-  #   max_num_concurrent_split_streams: 100  
+  #   max_num_concurrent_split_streams: 100
+
+  indices: []
```

Comment:
In quickwit terminology, we use `indexes` so it's better to stay consistent and use this word here.

Reply:
changed